server.js

```diff
@@ -129,6 +129,7 @@ function initialiseTemplateEngine (app) {
 }
 
 function initialisePublic (app) {
+  app.use('/.well-known/apple-developer-merchantid-domain-association.txt', express.static(path.join(__dirname, `/app/assets/apple-pay/${process.env.ENVIRONMENT}/apple-developer-merchantid-domain-association.txt`)))
```

Comment:
What happens if `ENVIRONMENT` isn’t set or is set to a value that doesn’t match one of the directories? Do we get a 404?

Reply:
Yep we get a 404